src/Tools/SemanticSearch/ReferenceAssemblies/ApiSet/System.Collections.Immutable.txt

```diff
@@ -0,0 +1 @@
+﻿
```

Comment:
what are these files?

Reply:
API filters. We used them to generate reference assemblies with filtered public surface. We can specify in each file which APIs to exclude.